README.md

```diff
@@ -29,7 +29,7 @@ You can use it as is without passing any option, or you can configure it as expl
   - `String` - set `origin` to a specific origin. For example if you set it to `"http://example.com"` only requests from "http://example.com" will be allowed.
   - `RegExp` - set `origin` to a regular expression pattern which will be used to test the request origin. If it's a match, the request origin will be reflected. For example the pattern `/example\.com$/` will reflect any request that is coming from an origin ending with "example.com".
   - `Array` - set `origin` to an array of valid origins. Each origin can be a `String` or a `RegExp`. For example `["http://example1.com", /\.example2\.com$/]` will accept any request from "http://example1.com" or from a subdomain of "example2.com".
-  - `Function` - set `origin` to a function implementing some custom logic. The function takes the request origin as the first parameter and a callback (which expects the signature `err [object], allow [bool]`) as the second, *async-await* and promises are supported as well.
+  - `Function` - set `origin` to a function implementing some custom logic. The function takes the request origin as the first parameter, `fastify` instance as a second and a callback (which expects the signature `err [object], allow [bool]`) as the last, *async-await* and promises are supported as well.
```

Comment:
I think this needs to be updated as well.

Reply:
Sorry, that file wasn't in commit for some reason. Added it as well.